server/telemetry/rudder.go

```diff
@@ -77,6 +77,10 @@ const (
 
 	eventChannelAction     = "channel_action"
 	actionRunChannelAction = "run_channel_action"
+
+	eventRunAction         = "run_action_event"
+	actionRunAction        = "run_action"
+	actionRunActionsUpdate = "run_actions_update"
```

Comment:
Nit: it's true that it's a bit of a mouthful, but if we want to be consistent with the other events and actions, this should be something like
```suggestion
	eventRunAction         = "run_action"
	actionRunAction        = "run_run_action"
	actionRunActionsUpdate = "update_run_actions"
```

Reply:
Maybe instead of `run_action` it could be `playbookrun_action`, so then it would be
```suggestion
	eventRunAction         = "playbookrun_action"
	actionRunAction        = "run_playbookrun_action"
	actionRunActionsUpdate = "update_playbookrun_actions"
```

Reply:
In any case, it's not that important, so I'll leave it up to you.

Reply:
I like this one, thanks.